_aggregations/pipeline/avg-bucket.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: default
+title: Average bucket
+parent: Pipeline aggregations
+nav_order: 10
+redirect_from:
+  - /query-dsl/aggregations/pipeline-agg#avg_bucket-sum_bucket-min_bucket-max_bucket/
```

Comment:
I am assuming this redirect is part of `sibling-aggregations`? Currently the indexed path is `https://docs.opensearch.org/docs/latest/aggregations/pipeline/index/#sibling-aggregations`. This does not really look consistent with it

Reply:
This is only needed for moved pages. Since this is a new page, removing the redirect.